faux-mgs/src/main.rs

```diff
@@ -289,14 +301,34 @@ enum Command {
         key: [u8; 4],
     },
 
+    /// Read a single key from the caboose
+    ReadComponentCaboose {
+        /// Component from which to read; must be `sp` or `rot`
+        #[clap(short, long, value_parser = parse_sp_component)]
+        component: SpComponent,
+
+        /// Target slot from which to read the caboose.
+        ///
+        /// The SP accepts `active` or `inactive`; the RoT accepts `A` or `B`
```

Comment:
I really like the specific targets here, but last time these were added, in `switch_default_image` (removed in favor of using existing calls like `set_component_active_slot`), both @jclulow and @jgallagher complained of the inconsistency with all the other commands that take slot directly as a `u16`. Personally, I'd prefer changing those to be more specific as well, but I"m not sure others feel the same.

Reply:
This is probably a "how much time do you spend with the RoT" question. Josh and I don't spend much with it, and are used to using 0/1 for host slots (even though those are also named A/B in some contexts) and 0 (vacuously) for SP updates. I'll make a suggestion to try to appease everyone in the match below.

Reply:
Hah, you couldn't see my comment because it was in draft until seconds ago, but that's my exact suggestion, yeah.

Reply:
Yeah, John agrees with that plan, so I did it below.